include/mvd/mvd_base.hpp

```diff
@@ -39,7 +40,8 @@ enum MVDType{
     UnknownFileType = 0,
     MVD2 = 20,
     MVD3 = 30,
-    Sonata = 40
+    Sonata = 40,
+    TSV = 50
```

Comment:
Hum... TSV is not an MVD format per se... more of an auxiliary file... I wouldn't register it here (unless it's required for some reason)